app/calculation/payment-rate-codes.js

```diff
@@ -0,0 +1,28 @@
+const paymentRateCodes = [
+  {
+    ambitionLevel: 'Introductory',
+    code: 'SS1A'
```

Comment:
I was wondering if it made sense to put the correct payment rates in here and use those rather than the ones in SA when providing the user with the different values for the ambition levels. But I assume when you PUT the selected ambition level, SA does an internal calculation with the stored payment rate so there will then be a mis-match. Assuming that is correct, we'll just have to go with the rates in SA.